### DISCUSSION TITLE
newTumbl : backing up of text under photos

### DISCUSSION THREAD

**Discussion by Pascal-0007** [TumblThreeApp/TumblThree#390] [Ideas]
First, let me tell you how happy I was today to be finally able to backup my blog on newTumbl. 

I tend to add comments under my photos. Other times, I may reblog posts because of the witty replies under photos. TumblThree backed up the photo just fine, but skipped all text underneath.

Would it be possible to backup the texts and replies accompanying an image or video post ?

**Comment by thomas694** [comment]
You can support TumblThree with a ⭐ 

What do you expect to happen with the text?
You can enable the option "download text posts" which downloads text elements to a separate file texts.txt. This feature is currently undergoing a rework.
You could also use the file rename functionality and add a title token. That way it "clues" this text directly to the image file.

**Reply by Zelarian** [reply]
I also want to save the captions under the photos, how do I use the file rename functionality and add a title token?
Edit: some text captions would be too long for a file name. The "download text posts" option is checked, but it only has the text only posts, not captions under photos.

**Reply by thomas694** [reply]
Write your desired format in the `filename template` field of the blog's settings. You can find help about the possible tokens in the tooltip or the wiki here.
If a title would be too long it's automatically shortened.

**Reply by Zelarian** [reply]
It didn't work. Newtumbl post titles are apparently a string of numbers, not the caption text in the post.
Edit: never mind, I'm dumb, I didn't realize the settings didn't automatically apply to the blogs when you change it, it applies to new blogs added. Thank you!
